_posts/2022-07-18-ols5-speedblog-open-science-mooc.md

```diff
@@ -0,0 +1,91 @@
+
+## Goals set at the beginning of the program
+
+Our main goal was to build a community around the Open Science MOOC, encouraging people to actively participate in our course and become active members of the Open Science community.
+
+## Key understanding and accomplishments
+
+The first key understanding for us was the concept of Minimum Viable Product (MVP) that we loved at first sight. It was incredibly useful to tone down our (sometimes excessive) expectations and concentrate on what we could really do based on time and resources available. Since the OSMOOC 2022 run started in the middle of the OLS program (week 7), we could not implement all the new tools (e.g. project management and community building). Using the MVP concept we decided to break down our goals, focus on making the course content more engaging for the current run, and make a plan on how we would implement bigger changes for next year's run.
+
+We also learned through the personas and pathways assignment to critically reflect on the audience we are trying to reach with the MOOC. Perhaps the way we structure the MOOC or the times that we give the MOOC does not necessarily fit the people we are trying to reach. For example, we run our course in May-June, and thinking from the perspective of a PhD candidate or support staff, we realized that this is one of the busiest times of the academic year, which leaves little time for taking on additional courses.
+
+A final and eye-opening understanding was that we could improve how the MOOC team works together, implementing open working practices such as contributors guidelines. Our team is made of people coming from different departments within TU Delft and working on the MOOC only for a part of the year.  We talked with Emma Karoune  who gave us great practical advice both on community building activities for the MOOC team and on implementing governance structure to improve communication, task divisions and responsibility.
+
+For me (Alessandra) learning how to use Github was perfect since I had it in my bucket list for a long time but never really found the time or motivation for it. For me (Lisanne) it was great to engage with fellow open scientists using Slack, LinkedIn and Twitter. I always wanted to start using social media for my work, and now I had a reason to.
+
+## The main goals achieved in this project
+
+### Engaging content
+
+When possible, we updated the course content making it more relevant and engaging. We expanded the historical introduction to the Open Science Movement, introducing more diverse elements, such as women or people outside Europe.
+
+![Historial illuminated manuscripts on parchment, with the title "Open Science: The beginning"](/images/2022-07-18-tudelft-history.png)
```

Comment:
To change the width, you can use `{: width="20%" }` write after the image, e.g.: `![Historial illuminated manuscripts on parchment, with the title "Open Science: The beginning"](/images/2022-07-18-tudelft-history.png){: width="20%" }`

Reply:
Thanks!